modules/javafx.graphics/src/main/java/javafx/application/Platform.java

```diff
@@ -432,4 +436,280 @@ public static ReadOnlyBooleanProperty accessibilityActiveProperty() {
+     *         <tr><td>{@code GTK.error_color}</td><td>{@link Color}</td></tr>
+     *         <tr><td>{@code GTK.success_color}</td><td>{@link Color}</td></tr>
+     *         <tr></tr>
+     *     </tbody>
+     * </table>
+     *
+     * @since 22
+     */
+    public interface Preferences extends ObservableMap<String, Object> {
+        /**
+         * The platform appearance, which specifies whether applications should use a light
+         * or dark color scheme. The value of this property is derived from the perceptual
+         * brightness of {@link #backgroundColorProperty() backgroundColor} in relation to
+         * {@link #foregroundColorProperty() foregroundColor} and defaults to {@link Appearance#LIGHT}
+         * if the platform does not report color preferences.
+         *
+         * @return the {@code appearance} property
+         */
+        ReadOnlyObjectProperty<Appearance> appearanceProperty();
+
+        /**
+         * Gets the value of the {@link #appearanceProperty() appearance} property.
+         *
+         * @return the platform appearance
+         */
+        Appearance getAppearance();
+
+        /**
+         * The color used for background regions.
+         * <p>
+         * If the platform does not report a background color, this property defaults to {@code Color.WHITE}.
+         *
+         * @return the {@code backgroundColor} property
+         * @defaultValue {@link Color#WHITE}
+         */
+        ReadOnlyObjectProperty<Color> backgroundColorProperty();
+
+        /**
+         * Gets the value of the {@link #backgroundColorProperty() backgroundColor} property.
+         *
+         * @return the background color
+         */
+        Color getBackgroundColor();
+
+        /**
+         * The color used for foreground elements like text.
+         * <p>
+         * If the platform does not report a foreground color, this property defaults to {@code Color.BLACK}.
+         *
+         * @return the {@code foregroundColor} property
+         * @defaultValue {@code Color.BLACK}
+         */
+        ReadOnlyObjectProperty<Color> foregroundColorProperty();
+
+        /**
+         * Gets the value of the {@link #foregroundColorProperty() foregroundColor} property.
+         *
+         * @return the foreground color
+         */
+        Color getForegroundColor();
+
+        /**
+         * The accent color.
+         * <p>
+         * If the platform does not report an accent color, this property defaults to {@code #157EFB}.
+         *
+         * @return the {@code accentColor} property
+         * @defaultValue {@code #157EFB}
+         */
+        ReadOnlyObjectProperty<Color> accentColorProperty();
+
+        /**
+         * Gets the value of the {@link #accentColorProperty() accentColor} property.
+         *
+         * @return the accent color
+         */
+        Color getAccentColor();
+
+        /**
+         * Returns the {@link Integer} instance to which the specified key is mapped.
```

Comment:
I think you can use `@code` rather than `@link` since there is already a link generated to `Integer` in the method signature.